osbrain/proxy.py

```diff
@@ -277,7 +301,7 @@ def release(self):
         """
         self._pyroRelease()
 
-    def proxy(self, name, timeout=3.):
+    def proxy(self, name, timeout=3., running=True):
```

Comment:
Instead of `running=True` I think we should have a more consistent API and put `timeout=X.`.

Reply:
Leftover code. It doesn't really make sense with the new implementation.